src/App.jsx

```diff
@@ -173,7 +188,12 @@ function App() {
             <div className="flex-auto">
               <Editor
                 defaultLanguage="Rego"
-                defaultValue={`package test\nallow = false`}
+                defaultValue={loadFile("../examples/default/policy.rego")}
```

Comment:
For external files we need to put them under `/public` folder, or it won't be bundled when `npm run build`